### ISSUE TITLE
Notebooks - lists sorting

### ISSUE THREAD

**Issue by sconim** [quillpad/quillpad#77]
Allow notebooks to be manually re-sorted both, in **left menu** as well as **in settings**.

**Comment by groaking** [comment]
I made a commit just recently in https://github.com/groaking/quillpad/commit/45bf5263e2241becaa509b22993152088249bde2. It allows sorting notebooks in both the nav drawer and "Manage Notebooks" menu by adjusting the respective sorting settings in the settings menu.

It does not allow manual sorting, though. But I believe alphabetical (both ascending and descending) ordering of notebook list suffices our daily need.

**Comment by opk12** [comment]
Ugly workaround is to prepend a number (`0 cat name`) and sort by name.

For me, the first one or two entries should be the most frequently used categories; or the categories of notes taken in a hurry, when I do not have time to scroll and pick a category.

**Comment by kuchta** [comment]
@opk12 How can it be sorted by name? I can't find it anywhere in the app...

**Comment by opk12** [comment]
> @opk12 How can it be sorted by name? I can't find it anywhere in the app...

in the settings, below the date - time format options

**Comment by kuchta** [comment]
I see just settings for sorting Notebooks and Tags, not Lists, @opk12

**Comment by opk12** [comment]
@kuchta Yes, they're the options, this app calls Lists the Notebook list and the Tag list from the left panel. Please keep this ticket focused and ask questions on [Matrix](https://matrix.to/#/#quillpad_community:gitter.im) or [Gitter](https://gitter.im/quillpad/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge).

**Comment by kuchta** [comment]
That's actually something different that List in a Note I've meant. No options from settings reflect that. The list is sorted just by modification time...
